server/routes/approvingLicences/handlers/approvalCases.ts

```diff
@@ -41,7 +41,7 @@ export default class ApprovalCaseRoutes {
           prisonerNumber: c.nomisRecord.prisonerNumber,
           probationPractitioner: c.probationPractitioner,
           submittedByFullName: _.head(c.licences).submittedByFullName,
-          releaseDate,
+          releaseDate: releaseDate ? format(releaseDate, 'dd MMM yyyy') : 'not found',
```

Comment:
selectReleaseDate now returns a date or null which means we can now: `string -> date -> string` rather than
`string -> date -> string -> date -> string`

Reply:
this weird `not found` string is existing logic that has been pulled out of `selectReleaseDate`